Cargo.toml

```diff
@@ -21,8 +21,8 @@ amplify = "4.0.0"
 baid58 = "0.3.1"
 strict_encoding = "2.0.1"
 strict_types = "1.0.0"
-rgb-std = { version = "0.10.0", features = ["fs"] }
-rgb-wallet = { version = "0.10.0", features = ["fs"] }
+rgb-std = { git = "https://github.com/RGB-WG/rgb-wallet", branch = "master", features = ["fs"] }
+rgb-wallet = { git = "https://github.com/RGB-WG/rgb-wallet", branch = "master", features = ["fs"] }
```

Comment:
Nit: it is better to add this as a separate lines in `[patch.crates-io]` sections to simplify its removal.

Reply:
yes, much better